docs/api-docs/catalog/currencies/currencies-overview.md

```diff
@@ -37,41 +37,41 @@ In BigCommerce, there are two types of currency that can be set up: a **display
 <!-- theme: info -->
 
 ### Note
-> * If the shopper’s card-issuing bank differs from the merchant's bank account country, additional cross-border fees may apply that can change the final settlement amount.
+> * If the shopper's card-issuing bank differs from the merchant's bank account country, additional cross-border fees may apply that can change the final settlement amount.
 > * You can choose whether a currency should be display-only or transactional when adding or editing a currency.
 
 </div>
 </div>
 </div>
 
-## Pre-Configuring the Store
+## Pre-configuring the Store
 
 <a id="pre-configuring-the-store"></a>
 
-Before setting up multiple currencies on a store, ensure the store is configured as follows:
+Before setting up multiple currencies on a store, ensure to configure the store as follows:
 * Stripe Payment Gateway
 * Stencil Theme
 * Optimized One-Page Checkout
 
-Additionally, review the [Currency API reference](https://developer.bigcommerce.com/api-reference/store-management/currency-api) to get aquainted with the endpoints and resources related to configuring multiple currencies.
+Additionally, review the [Currency API reference](https://developer.bigcommerce.com/api-reference/store-management/currency-api) to get acquainted with the endpoints and resources related to configuring multiple currencies.
 
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
 <!-- theme: info -->
 
-> Make sure to not change the store’s default currency. This could cause currency to not function properly.
+> Make sure not to change the store’s default currency. This change could cause currency to not function properly.

```

Comment:
What does "This change could cause currency to not function properly..." mean?
This sentence needs some clarification.